tests/ManagedETWTests/EtwTestsCS.csproj

```diff
@@ -8,7 +8,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>EtwTestsCS</RootNamespace>
     <AssemblyName>EtwTestsCS</AssemblyName>
-    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
should we also clone the tests project so that we exercise the net6.0 version as well?